plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.scss

```diff
@@ -121,6 +121,35 @@
 			font-weight: 400;
 			line-height: $medium-gap;
 		}
+
+		&.woocommerce-marketplace__product-card--theme {
+			padding: 0;
+			overflow: hidden;
+
+			.woocommerce-marketplace__product-card__content {
+				grid-template-rows: auto 1fr auto;
+				grid-template-columns: 2fr 1fr;
+			}
+			.woocommerce-marketplace__product-card__image {
+				grid-column-start: span 2;
+			}
+			.woocommerce-marketplace__product-card__image-inner {
+				width: 100%;
+			}
+			.woocommerce-marketplace__product-card__header {
+				padding-left: $medium-gap;
+			}
+			.woocommerce-marketplace__product-card__price {
+				padding-right: $medium-gap;
+				text-align: right;
+				display: inline-flex;
+				align-items: flex-end;
+				line-height: 1.5;
```

Comment:
One of the children of this element, the `woocommerce-marketplace__product-card__price-billing` has a slightly different `line-height`, set to `$medium-gap`. So the `flex-end` doesn't quite align things perfectly. 

![image](https://github.com/woocommerce/woocommerce/assets/10389957/589cbda6-eb13-4c74-a051-82e46283b14c)

This looks a bit off because of this reason. I recommend making the line heights the same for `.woocommerce-marketplace__product-card__price` and `woocommerce-marketplace__product-card__price-billing`.

Reply:
Thanks I missed it! Updated it now [2a5e0a5](https://github.com/woocommerce/woocommerce/pull/40195/commits/2a5e0a5ee3523fa42ef6095deacd110bbd9b817e)